src/connectors/rating-menu/connectRatingMenu.js

```diff
@@ -114,23 +114,52 @@ export default function connectRatingMenu(renderFn, unmountFn) {
       throw new Error(usage);
     }
 
+    let createURLFn;
```

Comment:
The previous implementation didn't cached the function like `toggeRefinement`. On each render a new function is created. We can extract the body of the function and inline the call in both `init` & `render`.